tests/cases/manage_dbserver/vars.json

```diff
@@ -0,0 +1,84 @@
+{
+ "use_hostname": false,
+ "pg_data": "/opt/pgdata",
+ "pg_wal": "/opt/pgwal",
+ "pg_script_table": "test1",
+ "pg_query_table": "test2",
+ "pg_postgres_conf_params": [
```

Comment:
Tests for Postgres user management are missing (`pg_users` variable).

I think we can create a fresh new user, test if this user exists with the right attributes, and then use this user for `pg_databases` and `pg_grant_role`. This way, we won't have to update the `vars.json` file on the fly.
Thought?